File: report_school/lib/models/sekolah.dart
class Sekolah {
  final int? idSekolah;
  final String? namaSekolah;
  final String? lokasi;

  Sekolah({this.idSekolah, this.namaSekolah, this.lokasi});

  factory Sekolah.fromJson(Map<String, dynamic> json) {
    return Sekolah(
      idSekolah: json['id_sekolah'] as int?,
      namaSekolah: json['nama_sekolah'] as String?,
      lokasi: json['lokasi'] as String?,
    );
  }
}

File: backend/app/Models/Sekolah.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class Sekolah extends Model
{
    use HasFactory;
    protected $table = 'sekolah';
    protected $primaryKey = 'id_sekolah';
    protected $guarded = [];
    protected $fillable = ['nama_sekolah', 'lokasi'];
    public $timestamps = true;
}
